src/components/apps/utils.js

```diff
@@ -51,6 +51,21 @@ export const getAppLaunchPath = (systemId, appId, versionId) =>
 export const getSavedLaunchPath = (systemId, appId, launchId) =>
     `/${NavigationConstants.APPS}/${systemId}/${appId}/launch?saved-launch-id=${launchId}`;
 
+/**
+ * Builds a path to the direct instant launch page.
+ *
+ * @param {string} ilId The Instant Launch ID
+ * @param {string} resourcePath A file or folder path to pass to the instant launch (optional)
+ */
+export const getInstantLaunchPath = (ilId, resourcePath = "") => {
+    var uri = `/${NavigationConstants.INSTANT_LAUNCH}/${ilId}`;
```

Comment:
This can also be a `const`:
```suggestion
    const uri = `/${NavigationConstants.INSTANT_LAUNCH}/${ilId}`;
```

Reply:
Oh, right, I'm not actually modifying this string later, just returning a value with a suffix. Good catch!